corehq/apps/reports/static/reports/js/bootstrap5/standard_hq_report.js

```diff
@@ -23,7 +23,7 @@ hqDefine("reports/js/bootstrap5/standard_hq_report", [
         }
 
         if (typeof standardHQReport !== 'undefined') {
-            // Custom reports, notably ewsghana
+            // Custom reports
```

Comment:
Is this needed for anything other than ewsghana? If not, maybe it can be removed too?